pymc3/model.py

```diff
@@ -1116,8 +1116,6 @@ def register_rv(self, rv_var, name, data=None, total_size=None, dims=None, trans
             ):
                 raise TypeError("Observed data cannot consist of symbolic variables.")
 
-            data = pandas_to_array(data)
```

Comment:
Won't this cause problems if a `DataFrame` is passed?

Reply:
One of the first things in `make_obs_var` is another call to `pandas_to_array`, that's why this one can go away